examples/kubernetes/encryption_in_transit/README.md

```diff
@@ -18,21 +18,21 @@ spec:
     - ReadWriteOnce
   persistentVolumeReclaimPolicy: Retain
   storageClassName: efs-sc
-  mountOptions:
-    - tls
   csi:
     driver: efs.csi.aws.com
     volumeHandle: [FileSystemId] 
+    volumeAttributes:
+      encryptInTransit: "true"
 ```
-Note that encryption in transit is enabled using mount option `tls`. Replace `VolumeHandle` value with `FileSystemId` of the EFS filesystem that needs to be mounted.
+Note that encryption in transit is configured using volume attribute `encryptInTransit`. By default, encryption in transit is enabled and there is no need to set `encryptInTransit` true. Replace `VolumeHandle` value with `FileSystemId` of the EFS filesystem that needs to be mounted.
```

Comment:
I think you should add the table from the PR to this README file, I think its more clear than this description.

Reply:
PTAL. I just replaced it with "The following table illustrates how the setting of encryptInTransit determines whether encryption in transit is enabled or not: ..."